pkg/api/install/install.go

```diff
@@ -34,35 +34,55 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-// userResources is a group of resources mostly used by a kubectl user
-var userResources = []string{"rc", "svc", "pods", "pvc"}
-
 const importPrefix = "k8s.io/kubernetes/pkg/api"
 
 var accessor = meta.NewAccessor()
 
+// availableVersions lists all known external versions for this group from most preferred to least preferred
```

Comment:
does the order here actually affect anything? once registration is cleaned up, would you envision this becoming the default order?